spec/30-processing-model.md

```diff
@@ -0,0 +1,47 @@
+# Processing Model
+
+This section describes the behavior of a tracing system when a new request is received. 
+As stated above, tracing systems can decide for each request whether they want to just _forward_ or _participate_ in a trace context. Additionally, when participating, a system needs to be able to start or restart a trace.
+
+## Processing Model in Forwarding Mode
+
+In forwarding mode, a tracing system decides to forward tracing information but does not modify the trace-context headers. The request is then considered as not sampled.
```

Comment:
I don't understand your comment. We defined this mode for tracers that explicitly do not change the trace. We also had the wording participating and not-participating for a while.

Reply:
Concrete example: we used to use HAProxy as a routing layer. It did not alter the tracing headers, so according to definitions here it was in the forwarding mode. However, it was still checking the tracing headers, and if the sampling bit was on it would record certain timing events and submit them to the tracing backend, where they would be merged into the client-side span of the caller.

Reply:
Resolved in https://github.com/w3c/trace-context/pull/246/commits/4b61d5c458e64aeb581ee769e76c86f0a16dbbcd